### PR TITLE
[MIG] business_requirement_sale_timesheet: Migration to v12

### PR DESCRIPTION
**Pull request by martintorresvm**
cc @Tecnativa

### CODE REVIEW

**Comment by martintorresvm** [comment]
cc @Tecnativa

**Review by pedrobaeza — APPROVED** [review]

File: business_requirement_sale_timesheet/__init__.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Copyright 2019 Tecnativa Victor M.M. Torres>
```
Comment (Tardo) [inline business_requirement_sale_timesheet/__init__.py]:
```suggestion
# Copyright 2019 Tecnativa - Victor M.M. Torres
```

File: business_requirement_sale_timesheet/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Copyright 2019 Tecnativa Victor M.M. Torres>
```
Comment (Tardo) [inline business_requirement_sale_timesheet/__manifest__.py]:
```suggestion
# Copyright 2019 Tecnativa - Victor M.M. Torres
```

File: business_requirement_sale_timesheet/models/__init__.py [hunk 3]
```diff
@@ -0,0 +1,4 @@
+# Copyright 2019 Tecnativa Victor M.M. Torres>
```
Comment (Tardo) [inline business_requirement_sale_timesheet/models/__init__.py]:
```suggestion
# Copyright 2019 Tecnativa - Victor M.M. Torres
```

File: business_requirement_sale_timesheet/models/project_task.py [hunk 4]
```diff
@@ -0,0 +1,28 @@
+# Copyright 2019 Tecnativa Victor M.M. Torres>
```
Comment (Tardo) [inline business_requirement_sale_timesheet/models/project_task.py]:
```suggestion
# Copyright 2019 Tecnativa - Victor M.M. Torres
```

**Review by Tardo — APPROVED** [review]
LGTM :+1: Only minor, non-blocking, changes...

**Comment by martintorresvm** [comment]
@rafaelbn I stashed your commits with the migration one.

**Review by rafaelbn — APPROVED** [review]

**Pull request merged**